### DISCUSSION TITLE
Vue plugins

### DISCUSSION THREAD

**Discussion by Dontorpedo** [filamentphp/filament#4369] [General]
Hi, 
i try to port a app i made with laravel nova to filament, i have some custom vue plugins in there.. would it be possible to use those in filament or do i have to recreate them in livewire? they are littlebit complex, so i try to avoid recreating them XD

**Comment by danharrin** [comment]
Livewire doesn't have much active support for Vue, but you can have a go. We can't really hold your hand much though... I don't have experience with that integration
